tests/testthat/test_azimuth.R

```diff
@@ -0,0 +1,31 @@
+# Tests for azimuth
+set.seed(42)
+
+library(Seurat)
+library(SeuratData)
+library(Azimuth)
+
+
+reference <- LoadData("pbmcref", "azimuth")
+query <- LoadData("ifnb")
+query <- NormalizeData(query, verbose = FALSE)
+multilayer_query <- split(query, f = query$stim)
+
+context("RunAzimuth()")
+
+test_that("RunAzimuth handles multi-layer queries", {
+  # split query
+  annotated_query <- RunAzimuth(
+    query = multilayer_query,
+    reference = "pbmcref"
+  )
+  # integrated query
+  integrated_query <- RunAzimuth(
+    query = query,
+    reference = "pbmcref"
+  )
+  expect_equal(ncol(annotated_query), ncol(integrated_query))
+  expect_equal(nrow(annotated_query@meta.data), nrow(integrated_query@meta.data))
+  expect_equal(ncol(annotated_query@meta.data), ncol(integrated_query@meta.data))
+  expect_equal(dim(annotated_query[["ref.umap"]]), dim(integrated_query[["ref.umap"]]))
```

Comment:
These checks are _okay_ but probably ought to be stronger. These tests should help to convince us that `RunAzimuth` is implemented correctly—obviously, we care more about just the dimensions of the integrated query for this. 

The way I'd structure these tests is first to try to establish that `RunAzimuth` works as expected with a single-layer query. With that in place, you're existing test should check that the results differ from those obtained with a single-layer query. They won't be the strongest tests, but they should give us a reasonable amount of confidence.